application/src/main/java/com/github/pieter_groenendijk/model/DigitalProduct.java

```diff
@@ -0,0 +1,66 @@
+package com.github.pieter_groenendijk.model;
+
+import jakarta.persistence.*;
+
+import java.util.Date;
+
+@Entity
+@Table(name = "DigitalProduct")
+public class DigitalProduct extends Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long digitalProductId;
+
+    @Column(name = "title", nullable = false, length = 100)
+    private String accessStatus;
+
+    @Column(name = "releaseDate", nullable = true)
+    private Date releaseDate;
```

Comment:
Dus er in Product een **year** (als dat releaseYear betekent), en hier een **releaseDate**? 

**Zijn de genoemde attributen niet dubbelop?**